docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/ComponentControlsCodeSandbox.tsx

```diff
@@ -4,9 +4,18 @@ import CodeSandboxer from 'react-codesandboxer'
 import { ComponentSourceManagerLanguage } from 'docs/src/components/ComponentDoc/ComponentSourceManager'
 import { appTemplate } from './indexTemplates'
 import createPackageJson from './createPackageJson'
-import { Icon, Tooltip } from 'src/index'
+
+export enum CodeSandboxState {
+  Default = 'DEFAULT',
```

Comment:
```suggestion
  Idle = 'IDLE',
```

I think `idle` will be better